### ISSUE TITLE
[$40][MSFT-203] The NVDA doesn't read link in footer-menu when user try to mouse hover on them

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2867]
**Steps to Reproduce**
1. Navigate to https://www.topcoder.com/challenges?
2. Try to mouse hover on footer menu: ABOUT US/ CONTACT US/ HELP CENTER/ PRIVACY POLICY/ TERMS

**Expected Result**
The NVDA should read link in footer-menu when user try to mouse hover on them

**Actual Result**
The NVDA doesn't read link in footer-menu when user try to mouse hover on them

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.2.3 Audio Description or Media Alternative (Prerecorded)
**Screen Reader Used**: Other
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=b9c070b1%2Dbb96%2D40e1%2Db92f%2D16f47db5d911&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fdoremihong%2FTest%20for%20Accessibility%20Challenge%2FBug%5F16

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096159 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096159 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096159 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by crazyk07** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097857 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097857 has been updated - it has been assigned to gauravseta.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by r0hit-gupta** [comment]
@sushilshinde @mishacucicea unable to reproduce this.

Test URL - https://www.topcoder.com/challenges
Env - Windows/Chrome
Screen Reader Used - NVDA

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097857 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
QA reviewer, seems to be fixed already in prod, please confirm

**Comment by drasticdpk** [comment]
Verified with NVDA on.
Operating System: Windows
Browser: Chrome

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097857<br/><br/>```This is an automated message for crazyk via Topcoder X```
